Examples/src/main/java/com/mapbox/vision/examples/SafetyActivity.java

```diff
@@ -26,9 +31,25 @@
  */
 public class SafetyActivity extends BaseActivity {
 
-    private Float maxAllowedSpeed = -1f;
+    private float maxAllowedSpeed = -1f;
     private boolean visionManagerWasInit = false;
     private VisionView visionView;
+    private FrameLayout speedAlertView;
+    private TextView speedLimitValueView;
+    private ImageView speedLimitSignView;
+
+    private static class SpeedLimit {
+        private int imageResId;
+        private int textColorId;
+
+        SpeedLimit(final int imageResId, final int textColorId) {
+            this.imageResId = imageResId;
+            this.textColorId = textColorId;
+        }
+    }
+
+    private final SpeedLimit overspeeding = new SpeedLimit(R.drawable.speed_limit_overspeeding, android.R.color.white);
+    private final SpeedLimit normal = new SpeedLimit(R.drawable.speed_limit_normal, android.R.color.black);
```

Comment:
Is the `speed_limit_overspeeding` icon in this PR as well? Dont see it in the diff...

Reply:
Indeed, git did not add it because of renaming